test/react-native/component.test.js

```diff
@@ -21,7 +21,7 @@ describe('App', () => {
     const query = gql`query people { allPeople(first: 1) { people { name } } }`;
     const data = { allPeople: { people: { name: 'Luke Skywalker' } } };
     const networkInterface = mockNetworkInterface({ request: { query }, result: { data } });
-    const client = new ApolloClient({ networkInterface });
+    const client = new ApolloClient({ networkInterface, addTypename: false });
```

Comment:
Yeah I guess it just seems like the sort of change that is hard to back out later. But i'll merge it and do that later today.